rootston/cursor.c

```diff
@@ -305,11 +305,47 @@ static void roots_cursor_press_button(struct roots_cursor *cursor,
 	}
 }
 
+static void notify_relative_motion(struct roots_seat *seat, uint64_t time_msec,
+		double dx, double dy, double dx_unaccel, double dy_unaccel) {
+
+	struct wlr_relative_pointer_manager_v1 *relative_pointer_manager =
+		seat->input->server->desktop->relative_pointer_manager;
+
+	struct wlr_seat_client *client = seat->seat->pointer_state.focused_client;
+	if (client == NULL) {
+		return;
+	}
+
+	struct wl_resource *resource;
+	wl_resource_for_each(resource, &relative_pointer_manager->relative_pointers) {
+
+		struct wlr_relative_pointer_v1 *pointer =
+			wlr_relative_pointer_v1_from_resource(resource);
+		if (pointer == NULL || seat->seat != pointer->seat) {
+			continue;
+		}
+
+		wlr_relative_pointer_v1_send_relative_motion(pointer, time_msec,
+				dx, dy, dx_unaccel, dy_unaccel);
+	}
+
+	wl_resource_for_each(resource, &client->pointers) {
+		if (wlr_seat_client_from_pointer_resource(resource) == NULL) {
+			continue;
+		}
+		wl_pointer_send_frame(resource);
```

Comment:
This should not be done in rootston. This should probably be moved in `wlr_relative_pointer_v1_send_relative_motion`.